qcow2reader_test.go

```diff
@@ -22,10 +22,73 @@ const (
 	CompressionTypeNone = qcow2.CompressionType(255)
 )
 
-func BenchmarkRead(b *testing.B) {
+// Benchmark sparse image with 10% utilization.  This is the best case when we
+// don't have to read any cluster from storage.
+func BenchmarkRead0p(b *testing.B) {
```

Comment:
```suggestion
func BenchmarkRead10p(b *testing.B) {
```

Reply:
This is an error in the code - leftover from previous version. I wanted to test 0% utilization.

Reply:
The comment line should be updated?

Reply:
The comment is fixed now.